docs/infrastructure.md

```diff
@@ -0,0 +1,49 @@
+# Public testing infrastructure
+
+The public testing infrastructure is accessible at the following locations:
+
+<table border="1">
```

Comment:
There is better way to create a table in markdown if I could remember...

Reply:
Not in this case - ReadTheDocs can't deal with merged cells.